defaults/main.yml

```diff
@@ -18,3 +18,12 @@ drbd_interface: enp0s8
 drbd_network_shared_secret: wXE8MqVa
 
 drbd_vip: 192.168.250.100
+
+# Debian9
+drbd_additional_packages: ['drbd8-utils']
+# EPEL for RedHat, requires repo already configures, see geerlingguy.repo-epel
+# drbd_additional_packages: ['drbd84-utils', 'kmod-drbd84']
```

Comment:
We could change this to `drbd_redhat_additional_packages` for RedHat based yum packages. This would allow the flexibility to not have to define this variable differently between Debian and RHEL based OS's.